### PR TITLE
add missing creds types to tower_credential module

### PR DESCRIPTION
**Pull request by bbayszczak**
##### SUMMARY
Add the following missing native credentials types to the tower_credential module
 - 'satellite': 'Red Hat Satellite 6' # renaming of satellite6
 - 'aim': 'CyberArk AIM Central Credential Provider Lookup'
 - 'conjur': 'CyberArk Conjur Secret Lookup'
 - 'hashivault_kv': 'HashiCorp Vault Secret Lookup'
 - 'hashivault_ssh': 'HashiCorp Vault Signed SSH'
 - 'azure_kv': 'Microsoft Azure Key Vault'
 - 'kubernetes_bearer_token': 'OpenShift or Kubernetes API Bearer Token'
 - 'github_token': 'GitHub Personal Access Token'
 - 'gitlab_token': 'GitLab Personal Access Token'

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
tower_credential

##### AWX VERSION
```
awx: 14.0.0
```

### CODE REVIEW

**Comment by AlanCoding** [comment]
These are used for a deprecated field, and there's another way to use the module via the `credential_type` field.

**Comment by john-westcott-iv** [comment]
We should decide in general what are plans for deprecated fields are. Are we going to keep them current up to the deprecation point or just let them slowly go out of sync (like this one).

File: awx_collection/plugins/modules/tower_credential.py
Comment (AlanCoding) [inline awx_collection/plugins/modules/tower_credential.py]: this does fail some checks with

```
plugins/modules/tower_credential.py:275:10: E231: missing whitespace after ':'
```

**Comment by ryanpetrello** [comment]
@AlanCoding @john-westcott-iv I know this is deprecated, but this does seem like a fairly harmless thing to merge.  Any arguments against it?

**Comment by AlanCoding** [comment]
I would be okay with that, but we still need the whitespace issue fixed before we can merge.

**Comment by beeankha** [comment]
@bbayszczak Could you please push a commit with the whitespace addition? I tested your changes locally and everything looks good otherwise.

**Comment by beeankha** [comment]
@bbayszczak Never mind, I was able to go ahead and add this small edit.

**Review by beeankha — APPROVED** [review]

**Pull request merged**
